app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1580,20 +1576,18 @@ class BrowserTabFragment :
     }
 
     private fun configureWebViewForAutofill(it: DuckDuckGoWebView) {
-        if (deviceAuthenticator.hasValidDeviceAuthentication()) {
-            browserAutofill.addJsInterface(it, autofillCallback)
+        browserAutofill.addJsInterface(it, autofillCallback)
```

Comment:
This now means that the JS interface and result listeners are added regardless if the feature is enabled. I think this is OK since they wouldn't be called anyway if the feature is disabled. 

This just means that we won't have to add these things when the user enables (adds device auth, becomes an internal user or enables the feature through settings) the autofill feature since it is already there.